README.md

```diff
@@ -1,2 +1,102 @@
-# base-template
-Base repo template 
+# SSEC-JHU <package_name>
+
+[![CI Status](https://github.com/ssec-jhu/base-template/workflows/CI/badge.svg?branch=main)](https://github.com/ssec-jhu/base-template/actions)
+[![Documentation Status](https://readthedocs.org/projects/base-template/badge/?version=latest)](http://ssec-jhu-base-template.readthedocs.io/en/latest/?badge=latest)
+[![codecov](https://codecov.io/gh/ssec-jhu/base-template/branch/main/graph/badge.svg?token=0KPNKHRC2V)](https://codecov.io/gh/ssec-jhu/base-template)
```

Comment:
These can all be navigated to and are live links. Their status currently reports garbage, i.e., "unknown" as they ref the main branch which'll get updated once this is merged.

Reply:
To view an example of the docs goto https://ssec-jhu-base-template--1.org.readthedocs.build/en/1/

Reply:
:)